admin_manual/installation/nginx.rst

```diff
@@ -205,71 +209,52 @@ In this example the webroot is located at
-      # topic first.
-      #add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload;" always;
-      #
+      # HSTS settings
       # WARNING: Only add the preload option once you read about
       # the consequences in https://hstspreload.org/. This option
       # will add the domain to a hardcoded list that is shipped
       # in all major browsers and getting removed from this list
       # could take several months.
-      add_header Referrer-Policy "no-referrer" always;
-      add_header X-Content-Type-Options "nosniff" always;
-      add_header X-Download-Options "noopen" always;
-      add_header X-Frame-Options "SAMEORIGIN" always;
-      add_header X-Permitted-Cross-Domain-Policies "none" always;
-      add_header X-Robots-Tag "none" always;
-      add_header X-XSS-Protection "1; mode=block" always;
-
-      # Remove X-Powered-By, which is an information leak
-      fastcgi_hide_header X-Powered-By;
-
-      # Path to the root of your installation
-      root /var/www;
-
+      #add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload;" always;
+      
       location = /robots.txt {
           allow all;
           log_not_found off;
           access_log off;
       }
 
-      # The following 2 rules are only needed for the user_webfinger app.
-      # Uncomment it if you're planning to use this app.
-      #rewrite ^/.well-known/host-meta /nextcloud/public.php?service=host-meta last;
-      #rewrite ^/.well-known/host-meta.json /nextcloud/public.php?service=host-meta-json last;
+      location /.well-known {
+          # The following 6 rules are borrowed from `.htaccess`
 
-      # The following rule is only needed for the Social app.
-      # Uncomment it if you're planning to use this app.
-      #rewrite ^/.well-known/webfinger /nextcloud/public.php?service=webfinger last;
+          rewrite ^/\.well-known/host-meta\.json  /nextcloud/public.php?service=host-meta-json    last;
+          rewrite ^/\.well-known/host-meta        /nextcloud/public.php?service=host-meta         last;
+          rewrite ^/\.well-known/webfinger        /nextcloud/public.php?service=webfinger         last;
+          rewrite ^/\.well-known/nodeinfo         /nextcloud/public.php?service=nodeinfo          last;
 
-      location = /.well-known/carddav {
-        return 301 $scheme://$host:$server_port/nextcloud/remote.php/dav;
-      }
-      location = /.well-known/caldav {
-        return 301 $scheme://$host:$server_port/nextcloud/remote.php/dav;
-      }
-
-      location /.well-known/acme-challenge { }
-
-      location ^~ /nextcloud {
+          location /.well-known/carddav   { return 301 /nextcloud/remote.php/dav/; }
+          location /.well-known/caldav    { return 301 /nextcloud/remote.php/dav/; }
 
+          try_files $uri $uri/ /nextcloud/index.php$request_uri;
+      }
+      
+      location ~ ^/nextcloud($|/) {
```

Comment:
Sorry for not commenting on this previously, I didn't notice it. Why did you change this to a regex match instead of using the `^~` modifier?

Reply:
Woops, my recollection of the location nesting rules must've been wrong when I wrote this, will change this to `location ^~ /nextcloud`.